lib/module.js

```diff
@@ -484,12 +484,32 @@ function tryModuleLoad(module, filename) {
   }
 }
 
-Module._resolveFilename = function(request, parent, isMain) {
+Module._resolveFilename = function(request, parent, isMain, options) {
   if (NativeModule.nonInternalExists(request)) {
     return request;
   }
 
-  var paths = Module._resolveLookupPaths(request, parent, true);
+  var paths;
+
+  if (typeof options === 'object' && options !== null &&
+      Array.isArray(options.paths)) {
+    paths = [];
+
+    for (var i = 0; i < options.paths.length; i++) {
+      const path = options.paths[i];
+      const lookupPaths = Module._resolveLookupPaths(path, parent, true);
+
+      if (paths.includes(path) === false)
```

Comment:
I'm not sure it matters to compare strictly with a boolean like this like we do for other special values? /cc @bmeurer

Reply:
This will generate the same code as `if (!paths.includes(path))`, since TurboFan knows that `Array#includes` returns a boolean.

Reply:
Ah ok, then yes, let's switch these to the `if (!paths.includes(path))` style here and elsewhere.

Reply:
@mscdex done.